app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/LocalLibraryFragment.kt

```diff
@@ -186,11 +190,30 @@ class LocalLibraryFragment : BaseFragment() {
 
   override fun onOptionsItemSelected(item: MenuItem): Boolean {
     when (item.itemId) {
-      R.id.get_zim_nearby_device -> activity?.start<LocalFileTransferActivity>()
+      R.id.get_zim_nearby_device -> navigateToLocalLibraryFragment()
     }
     return super.onOptionsItemSelected(item)
   }
 
+  private fun navigateToLocalLibraryFragment() {
```

Comment:
seems like the wrong name for this method

Reply:
It really does...